### DISCUSSION TITLE
Customisable transpilation options

### DISCUSSION THREAD

**Discussion by thatvideoshopguy** [LangTrans/LangTrans#30] [Ideas]
Allow users to customise the transpilation process with various options, such as output formatting, inline comments preservation, and error handling strategies. This would give users more control over the transpilation output and allow them to tailor it to their specific requirements.

**Comment by B-R-P** [comment]
@thatvideoshopguy Thank you for your ideas

**Comment by thatvideoshopguy** [comment]
Pleasure @B-R-P, I'd be keen to assist in some of the discussion and issue points I raised. Also can discuss more in Discord, if you prefer.
